benchmarks/bayesmark/optuna_optimizer.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Union
+
+import numpy as np
+
+from bayesmark.abstract_optimizer import AbstractOptimizer
+from bayesmark.experiment import experiment_main
+import optuna
+from optuna import pruners
+from optuna import samplers
+
+
+_SAMPLERS = {
```

Comment:
I don't want to restrict the sampler in the current ones. I mean this benchmark should be feasible for any samplers we add to `optuna.samplers` in the future. How about instantiating the sampler class from its name as follows?

```python
if sampler_name in optuna.samplers.__dict__:
    try:
        if self.param.sampler in ["BoTorchSampler", "PyCmaSampler"]:
            sampler = getattr(optuna.integration, self.param.sampler)(**sampler_kwargs)
        else:
            sampler = getattr(optuna.samplers, self.param.sampler)(**sampler_kwargs)
        assert isinstance(sampler, optuna.samplers.BaseSampler)
    except TypeError:
        raise TypeError(
            f"__init__() of {self.param.sampler} got an unexpected keyword "
            f"argument. Given arguments are {sampler_kwargs}."
        )
```

Reply:
Sounds reasonable. I agree with the current design as you explained.